appstudio-utils/util-scripts/check-ec-policy.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/bash
+set -euo pipefail
+
+EC_WORK_DIR=${EC_WORK_DIR:-/tmp/ecwork}
+
+DATA_DIR=$EC_WORK_DIR/data
+POLICY_DIR=$EC_WORK_DIR/policies
+
+FORMAT=${1:-pretty}
+
+OPA_QUERY=${OPA_QUERY:-data.hacbs.contract.main.deny}
```

Comment:
Is there a way to set this? Would it make sense to make a param in the task that can toggle this?

Reply:
For now I'd prefer to keep it as an environment var which is almost as good as a bash parameter. In some ways it's better since you don't have to worry about the param ordering.